case-server/src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -134,4 +135,16 @@
         caseService.deleteAllCases();
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/cases/search")
+    @ApiOperation(value = "Search cases by metadata")
+    public ResponseEntity<List<CaseInfos>> searchCases(@RequestParam(value = "query") String query) {
+        LOGGER.debug("search cases request received");
+        List<CaseInfos> cases = caseService.searchCases(query);
+        if (cases == null) {
```

Comment:
don't test for null and don't return 204. It will just crash the clients who expects a list (even empty ?)

It seems like caseService doesn't even return null, which is good

Reply:
done